src/apps/content-editor/src/app/components/ReorderNav/DragComponents/DraggableItem.js

```diff
@@ -3,18 +3,16 @@ import React, { Component } from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faArrowsAlt, faCaretRight } from "@fortawesome/free-solid-svg-icons";
 
-import { AppLink } from "@zesty-io/core/AppLink";
-
 import styles from "./styles.less";
 export default class DraggableItem extends Component {
   render() {
-    const { ZUID, label, children, handleNestChange } = this.props;
+    const { ZUID, label, children, handleNestChange, ...rest } = this.props;
     return (
-      <li {...this.props}>
+      <li {...rest}>
         <FontAwesomeIcon icon={faArrowsAlt} />
 
         {children.length ? (
-          <AppLink
```

Comment:
What is the reason we are choosing to not use an `AppLink` component here?

Reply:
The draggable component cannot recognize the  AppLink `a` tag thus causing on drag to crash the app.  The span allows the the element to be nested in the list parent and reorder/drag works properly.